api/python/provisioner/commands/auto_deploy.py

```diff
@@ -55,19 +51,24 @@ class AutoDeploy(SetupCmdBase, CommandParserFillerMixin):
 
     @staticmethod
     def deployment_validations(deploy_check):
-        checker = Check()
+        check_import = import_module(
```

Comment:
I guess it won't work since all commands are exported at the very beginning  when api spec yaml is parsed: https://github.com/Seagate/cortx-prvsnr/blob/2dd21bdae0fcc26471cf56398df59ccf0b701ac1/api/python/provisioner/commands/__init__.py#L1563-L1569.
I guess you may try `importlib.reload` instead.

Reply:
Thanks @andkononykhin . 
`import_module()` did not reload and accept the new imports, `reload()` worked. Comment addressed.

Reply:
Why can't we do `reload` inside of `provisioner.commands.checks`? 
Also, we use the global variable `cortx_py_utils_import_error` in `checks` module, so we can validate if imports from `cortx.utils` was successful after `reload` in `deployment_validations` method.